Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -361,7 +361,7 @@ final class AttributeTests: ParserTestCase {
 
       assertParse(
         "@_implements(1️⃣\(baseType), f())",
-        AttributeSyntax.parse,
+        { AttributeSyntax.parse(from: &$0) },
```

Comment:
Does this actually fix things, or does each closure have to be annotated with `@Sendable`?

Reply:
It does fix it. Sendable is automatically inferred.